app/components/ui/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "jsx-a11y/heading-has-content": "off"
```

Comment:
because this directory will be a base (primitive) component, it'll never have content